packages/panels/resources/views/components/avatar/user.blade.php

```diff
@@ -4,6 +4,7 @@
 
 <x-filament::avatar
     :src="filament()->getUserAvatarUrl($user)"
+    :alt="__('filament-panels::layout.top_bar.avatar', ['user' => filament()->getUserName($user)])"
```

Comment:
Can you refactor the translation to use `filament-panels::layout.avatar.alt` please?